q2_moshpit/busco/busco.py

```diff
@@ -141,16 +154,59 @@ def _visualize_busco(output_dir: str, busco_results: pd.DataFrame) -> None:
         os.path.join(output_dir, "busco_results.csv"),
         index=False
     )
+    # Outputs different df for sample and feature data
     busco_results = _parse_df_columns(busco_results)
-    dfs = _partition_dataframe(busco_results, max_rows=100)
+    n = 100  # Max number of rows
+
+    if len(busco_results["sample_id"].unique()) >= 2:
```

Comment:
In this function, we don't have access to the input artifact, only to the results that Busco returned. Therefore we can't call `isinstance` because there is nothing to call it on in the first place.

Reply:
Ohhh sorry, GitHub tricked me into thinking it was part of another function by folding the code 🤦

Reply:
No worries!